### DISCUSSION TITLE
Filter out !mp commands in tournament mode

### DISCUSSION THREAD

**Discussion by LeoFLT** [ppy/osu#17367] [Ideas] [answered]
Currently, the chat for spectating tournament matches filters out BanchoBot responses to !mp commands, but the commands themselves are left as is. This tends to clutter the chat with unnecessary messages.

![image](https://i.imgur.com/MAPx1iQ.png)

Filtering these messages (and maybe commands like !stats and !where as well) would improve the viewing experience.

**Comment by peppy** [comment]
I'm not against doing this. The only reason I haven't until now is that I thought it felt pretty nice in terms of transparency to know what's going on behind the scene. Open to more opinions.

**Reply by cl8n** [reply]
printing a more readable alternative could be nice

e.g.
"LeoFLT: !mp team #{id} blue" -> "*LeoFLT switched {player} to the blue team*"
"LeoFLT: !mp timer 120" -> "*LeoFLT started a timer for 120 seconds*"

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/31150 simple way for now
